cmd/osctl/cmd/version.go

```diff
@@ -46,6 +46,9 @@ var versionCmd = &cobra.Command{
 
 				if resp.Metadata != nil {
 					node = resp.Metadata.Hostname
+					if resp.Metadata.Error != "" {
```

Comment:
this is just for debugging to see actual error from backend, should be implemented on top level for real debugging